doc/developer-notes.md

```diff
@@ -788,19 +788,19 @@ RPC interface guidelines
 
 A few guidelines for introducing and reviewing new RPC interfaces:
 
-- Method naming: use consecutive lower-case names such as `getrawtransaction` and `submitblock`
+- Method naming: use consecutive lower-case names such as `getrawtransaction` and `submitblock`.
 
   - *Rationale*: Consistency with existing interface.
 
-- Argument naming: use snake case `fee_delta` (and not, e.g. camel case `feeDelta`)
+- Argument naming: use snake case `fee_delta` (and not, e.g. camel case `feeDelta`).
 
   - *Rationale*: Consistency with existing interface.
 
 - Use the JSON parser for parsing, don't manually parse integers or strings from
   arguments unless absolutely necessary.
 
   - *Rationale*: Introduces hand-rolled string manipulation code at both the caller and callee sites,
-    which is error prone, and it is easy to get things such as escaping wrong.
+    which is error prone, and it is easy to get things, such as escaping, wrong.
```

Comment:
I prefer without the extra commas here.

Reply:
how about?:
which is error prone and hard to use. For instance, it is easy to incorrectly escape.